src/main/java/billgates/view/gui/ActionPanel.java

```diff
@@ -215,7 +217,48 @@ private void backFromSplit() {
     }
 
     private void addEntry() {
-        // TODO: Call the controller of InsertEntryUseCase
+        JTextField dateField = new JTextField(20);
+        JTextField valueField = new JTextField(20);
+        JTextField currencyField = new JTextField(20);
+        JTextField descriptionField = new JTextField(20);
+        JTextField fromField = new JTextField(20);
+        JTextField toField = new JTextField(20);
+        JTextField locationField = new JTextField(20);
```

Comment:
Some of these fields needs to align with the information that we have setup in the database. 

For reference: 
<img width="476" alt="image" src="https://user-images.githubusercontent.com/8016180/205409664-8642d01f-b147-4460-80af-9f3410fc637a.png">

Value should have length of 16, currency should have length of 3. 

All the rest might go over 20 characters, but I'm not really sure if we can specify them to be unlimited when initializing?

Reply:
GUI stuff doesn't affect database stuff.

`new JTextField(20)` is kind of like the length of the input field is approximately 20 characters.